app/views/node_info/_statistics.haml

```diff
@@ -0,0 +1,21 @@
+-#   Copyright (c) 2010-2011, Diaspora Inc.  This file is
```

Comment:
Technically, the header has to include a AGPL notice. But let's not make this part of this PR :)